tests/dragonfly/utility.py

```diff
@@ -560,3 +561,15 @@ def create(self, **kwargs):
 
 async def disconnect_clients(*clients):
     await asyncio.gather(*(c.connection_pool.disconnect() for c in clients))
+
+
+def gen_cert(ca, req, key, cert):
```

Comment:
use more descriptive names
gen_cert rename to gen_certificate
key rename to private_key_path
cert rename to certificate_path
req rename to certificate_request_path
also in this function I think it will be more readable if you path cs_key_path and cs_certificate_path and not ca dictionary